runtime/src/jvmTest/README.md

```diff
@@ -8,7 +8,7 @@ To generate the proto code for Kotlin:
 
 If on Windows, add the following arg after the `--kotlin_out` arg:
```

Comment:
Update `kotlin_out` to `pbandk_out` here and a few lines above

Reply:
I fixed by commit b6ee56e